Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```diff
@@ -397,6 +397,31 @@
     healthPerWeakeningeHit: -0.1
     stabilityPerWeakeningeHit: -0.2
 
+- type: entity
+  parent: AnomalousParticleDelta
+  id: AnomalousParticleOmegaStrong
+  noSpawn: true
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Projectiles/magic.rsi
+    layers:
+      - state: magicm_green
+        shader: unshaded
+  - type: AnomalousParticle
+    particleType: Default
+    anomalyStabilizer: true
+    anomalyWeakener: true
```

Comment:
if you include both of these, they'll balance out. Just remove the weakening effect.

Reply:
will do

Reply:
done